docs/.vuepress/plugins/dump-docs-data/docs-versions.js

```diff
@@ -22,6 +17,17 @@ async function readFromDocsLatest() {
   const response = await fetch('https://handsontable.com/docs/data/common.json');
   const data = await response.json();
 
+  // TODO: Mocked.
+  data.patches = [
```

Comment:
This mock causes happy tests on localhost. But on production/staging, there would be some problems. Can you remove that mock?

Reply:
Removed within https://github.com/handsontable/handsontable/pull/10079/commits/e539c2a3a296ae4735d921341bed747ab79092d7 and https://github.com/handsontable/handsontable/pull/10079/commits/1f924816080aa8ce52bd36efb36c252675e92e20.